css/main.css

```diff
@@ -122,34 +126,39 @@
   color: var(--color-error);
 }
 
+/* File drag and drop */
+
+.filesListDragDropNotice {
+  width: 100%;
+  min-height: 113px;
+  margin: 0;
+  user-select: none;
+  color: var(--color-text-maxcontrast);
+  background-color: var(--color-main-background);
+  border-color: #000;
+}
+
+.filesListDragDropNoticeWrapper {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  height: 15vh;

```

Comment:
De sleep bestand section ziet er niet uit
![image](https://github.com/user-attachments/assets/db859a29-933e-4c84-87dd-29416586faf1)

Verander height naar `fit-content` in plaats van een hard coded height

Reply:
Nice catch!